src/sentry/integrations/bitbucket/integration.py

```diff
@@ -26,7 +26,7 @@
 from sentry.utils.http import absolute_uri
 
 from .client import BitbucketApiClient
-from .issues import BitbucketIssueBasicMixin
+from .issues import BitbucketIssuesSpec
```

Comment:
i'm not satisfied with the names of these things 🤔

Reply:
`BitbucketIssueBasicMixin`/`BitbucketIssuesSpec` exists only to be inherited by `BitbucketIntegration`, right? Could we just delete it and move all of its contents directly into `BitbucketIntegration`? Then we wouldn't need to name it at all.

The main argument against doing that would be that it makes `BitbucketIntegration` too long. There is some value in keeping all the issue-related methods grouped together, I suppose.

But the difficulty in thinking of a name for the class can be a signal that the class doesn't represent something sensible and therefore shouldn't exist. 😉

Reply:
i'll keep this for now, but it's easily separable